src/components/content/order/common/ScaleOrModifySubmitStatusAlert.tsx

```diff
@@ -0,0 +1,128 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ */
+
+import {
+    ApiError,
+    DeployedServiceDetails,
+    Response,
+    ServiceProviderContactDetails,
+} from '../../../../xpanse-api/generated';
+import React, { useMemo } from 'react';
+import { useStopwatch } from 'react-timer-hook';
+import { ProcessingStatus } from '../orderStatus/ProcessingStatus';
+import { OperationType } from '../types/OperationType';
+import { OrderSubmitResult } from '../orderStatus/OrderSubmitResult';
+import { convertStringArrayToUnorderedList } from '../../../utils/generateUnorderedList';
+
+function ScaleOrModifySubmitStatusAlert({
```

Comment:
always the previous modification alert is displayed first when clicked on scale/modify